Lecture 3/Task2/CompareStudents.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Comparator;
+
+public class CompareStudents implements Comparator<Student> {

```

Comment:
Name of the class should reflect it's purpose. CompareStudents doesn't reflect which way comparasion is done. AgeCompareStudents works better in this case.